doc/coq.rst

```diff
@@ -114,9 +116,10 @@ The semantics of the fields are:
   plugins. Starting with ``(lang coq 0.6)``, ``<ocaml_plugins>`` must contain
   public library names.
 
-- Your Coq theory can depend on other theories by specifying them in the
-  ``<coq_theories>`` field. Dune then passes to Coq the corresponding flags for
-  everything to compile correctly (this corresponds to the ``-Q`` flag for Coq).
+- Your Coq theory can depend on other theories, installed or in your workspace,
```

Comment:
Just a question about terminology here, not sure we want to say "globally installed" to make things clearer.